### ISSUE TITLE
Audit writing style

### ISSUE THREAD

**Issue by tildechris** [cmu-delphi/www-covidcast#634]
In #625 it was noted that we have stylistic inconsistencies with regard to labeling.  @ryantibs had the idea to review the writing style everywhere.

As part of this audit, we should explicitly pick a style guide (AP, MLA, et. al.) so that it can be applied to all visualizations and blog content.

**Comment by Akvannortwick** [comment]
Hi, sorry this took me a bit. I wanted to review some information on AP before I went through the content and then wanted to tackle the API documentation for the upcoming release. 

While I focused on AP, I am not particular to it (I've used AP most recently which is why I chose it) and can use another style guide if someone else has any preference. 

I set up a [google doc](https://docs.google.com/document/d/1vr5jt5UDBUkf1lpVAvKN6GzSLf7iK7WCOW8ayVw5ml0/edit#) for now since these are preliminary findings and suggestions. As it grows I can move content to a specific area or keep it here.
